routes/users.ts

```diff
@@ -4,6 +4,7 @@ import authenticate from "../middlewares/authenticate";
 
 const router = Router();
 
+// eslint-disable-next-line
 router.get("/self", authenticate, getSelfData);
```

Comment:
Could we add a todo to check why we are adding this and how to mitigate that

Reply:
Thanks for mentioning this @harshith-venkatesh add a Todo comment.